frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -492,6 +512,18 @@ const plugin: Plugin<ConsumedExtensions> = [
         )).default,
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      perspective: 'dev',
+      exact: false,
+      path: ['/dev/monitoring/all-namespaces', '/dev/monitoring/ns/:ns'],
```

Comment:
> Selecting namespace form the project selector doesn't update the URL. Similarly, changing the namespace in the URL doesn't update the selected item in the project selector.

This issue is happening because of ```dev``` in the path. Do we need ```dev``` in the path?

Reply:
I think `dev` has been added in the path because there's already a `Monitoring` Nav item present in the admin perspective (which has Alerting, Metrics & Dashboards as sub-items).

Reply:
As @debsmita1 mentioned, I had added `dev` in path to differentiate `Monitoring` Nav in Admin & Dev perspective. But having `/dev` in path doesn't update the URL on namespace change & hence fixed the bug by changing path with dev prefix.